homeassistant/components/knx/config_flow.py

```diff
@@ -199,52 +208,145 @@ async def async_step_import(self, config: dict | None = None) -> FlowResult:
         if self._async_current_entries() or not config:
             return self.async_abort(reason="single_instance_allowed")
 
+        data = {
+            ConnectionSchema.CONF_KNX_RATE_LIMIT: config.get(
+                ConnectionSchema.CONF_KNX_RATE_LIMIT,
+                ConnectionSchema.CONF_KNX_DEFAULT_RATE_LIMIT,
+            ),
+            ConnectionSchema.CONF_KNX_STATE_UPDATER: config.get(
+                ConnectionSchema.CONF_KNX_STATE_UPDATER,
+                ConnectionSchema.CONF_KNX_DEFAULT_STATE_UPDATER,
+            ),
```

Comment:
```suggestion
            ConnectionSchema.CONF_KNX_RATE_LIMIT: config[
                ConnectionSchema.CONF_KNX_RATE_LIMIT
            ],
            ConnectionSchema.CONF_KNX_STATE_UPDATER: config[
                ConnectionSchema.CONF_KNX_STATE_UPDATER
            ],
```
these have defaults set. Why do we `get` these but not `ConnectionSchema.CONF_KNX_MCAST_GRP`?

Reply:
Because those configs dont come from the YAML, they come from the config entry and can, depending on what type of flow the user has finished, be None

Reply:
Makes sense for those :)